modules/videoio/src/videoio_registry.cpp

```diff
@@ -175,14 +180,25 @@ static const struct VideoBackendInfo builtin_backends[] =
                            0)
 #endif
 #endif
-
 #ifdef HAVE_OBSENSOR
     DECLARE_STATIC_BACKEND(CAP_OBSENSOR, "OBSENSOR", MODE_CAPTURE_BY_INDEX, 0, create_obsensor_capture, 0)
 #endif
-
     // dropped backends: MIL, TYZX
 };
 
+static const struct VideoDeprecatedBackendInfo deprecated_backends[] =
+{
+#ifdef _WIN32
+    DECLARE_DEPRECATED_BACKEND(CAP_VFW, "Video for Windows")
+#endif
+    DECLARE_DEPRECATED_BACKEND(CAP_QT, "QuickTime")
+    DECLARE_DEPRECATED_BACKEND(CAP_UNICAP, "Unicap")
+    DECLARE_DEPRECATED_BACKEND(CAP_OPENNI, "OpenNI")
+    DECLARE_DEPRECATED_BACKEND(CAP_OPENNI_ASUS, "OpenNI")
+    DECLARE_DEPRECATED_BACKEND(CAP_GIGANETIX, "GigEVisionSDK")
```

Comment:
We don't need macro here, see https://github.com/opencv/opencv/commit/872bb3255a9d87ee4be784b84fd30226a34d911e